hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/bootstrap/BootstrapOperator.java

```diff
@@ -130,7 +130,7 @@ public void initializeState(StateInitializationContext context) throws Exception
     }
 
     this.hadoopConf = HadoopConfigurations.getHadoopConf(this.conf);
-    this.writeConfig = FlinkWriteClients.getHoodieClientConfig(this.conf, true);
+    this.writeConfig = FlinkWriteClients.getHoodieClientConfig(this.conf, false, true, true);
```

Comment:
why this change?

Reply:
We've changed the default record merger from HoodieAvroRecordMerger to EventTimeFlinkRecordMerger to support rowData writing for COW, but rowData regular reader has not been supported yet, so here add a new parameter to indicate whether the writeConfig is created for regular reader.

After rowData regular reader is supported, we can remove the indicator parameter. And related note has been added [here](https://github.com/apache/hudi/blob/1df6c10a5dd2a7e931f57cadbd97b749fcf40909/hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/util/FlinkWriteClients.java#L242)